src/System Application/Test/Azure File Services API/src/AFSTestHelper.Codeunit.al

```diff
@@ -0,0 +1,56 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.Azure.Storage.Files;
+
+using System.Azure.Storage;
+
+codeunit 132515 "AFS Test Helper"
```

Comment:
This name doesn't tell me what the code does... I'd name this 
AFS Get Test Storage Authorization

or a variation/abbreviation thereof... 

also, just FYI, I very much frown on terms like Helper and Management because they do not add anything. You might as well not add them to the name and you'd get the same amount of info... I think we can name things aptly with a little effort.

Reply:
Okay, I agree. Changed to AFS Get Test Storage Auth.